server/app/assets/javascripts/admin_programs.ts

```diff
@@ -26,6 +27,21 @@ class AdminPrograms {
         cardsPlaceholder.classList.add('hidden')
       }
     }
+
+    const withCopyableProgramLink = Array.from(
+      cardsParent.querySelectorAll(
+        `[${AdminPrograms.PROGRAM_LINK_ATTRIBUTE}]`
+      ) as any as Array<HTMLElement>
+    )
+    withCopyableProgramLink.forEach((el) => {
+      const programLink = el.getAttribute(AdminPrograms.PROGRAM_LINK_ATTRIBUTE)
+      if (!programLink) {
+        return
```

Comment:
suggestion (blocking): log a warning that there's no program link where we expect it to be